packages/react/src/components/Tab/Tab-test.js

```diff
@@ -110,6 +110,20 @@ describe('Tab', () => {
       });
     });
   });
+
+  describe(' disabled toggles aria-disabled', () => {
```

Comment:
```suggestion
```

Reply:
Won't removing the `describe` mess things up? It seemed like the pattern was to put tests in one?

Reply:
Would you want this block of tests to be put in the `describe('renders as expected')` section?

Reply:
I moved it there, let me know if that works!